framework/test/gtest/BundleManifestTest.cpp

```diff
@@ -36,6 +36,59 @@
 
```

Comment:
this file should be formatted using the clang.format file in the root source directory